systemd/renderd.service

```diff
@@ -1,8 +1,12 @@
 [Unit]
-Description=Rendering daemon for Openstreetmap tiles
+Description=Daemon that renders map tiles using mapnik
+Documentation=man:renderd
+After=network.target auditd.service
 
 [Service]
-ExecStart=/usr/bin/renderd -f
+RuntimeDirectory=/run/renderd
+RuntimeDirectoryMode=0755
+ExecStart=/usr/local/bin/renderd -f
```

Comment:
If the deb.openbuildingmap.org package uses this service it will fail with the wrong path. Typically the service file get processed through the build system to have the correct path no matter the end binary location.

Reply:
Many thanks for the feedback!

The debs (in the temporary deb.openbuidlingmap.org or [Debian](https://packages.debian.org/bullseye/renderd) or [Ubuntu](https://packages.ubuntu.com/hirsute/renderd)) use the [`renderd.service` from the packaging](https://salsa.debian.org/debian-gis-team/libapache2-mod-tile/-/blob/master/debian/renderd.service). This PR is thought to include the `renderd.service` suitable when one compiles directly.

Do you think this is accurate then? Or should it be different then?

Reply:
Build systems are not my strong point, `/usr/local` may be the default, but it's not usually the case for end users. It is usually configurable at build time.

Reply:
Do you suggest a `ExecStart=renderd -f`?

Reply:
No, $PATH is not honoured in unit files.

Reply:
What would be the accurate line? Please help me out.

Reply:
I think you need to include it in your build system with something like `$(bindir)/renderd -f`, but this isn't my area, I don't really know what I'm doing.

Reply:
I see your point. And makes perfect sense. I build the debs with it and that is my use case. Personally, I don't need the `systemd.service` from compilation. Please anyone who wants to use the `systemd.service` unit from compiling, contribute to it.

I am considering closing the PR in the meantime, at least if no one comes forward within a reasonable time who needs it and can develop this PR further. Perhaps it is not necessary for `mod_tile` to provide this. Most people should use the packaged debs anyway.

Reply:
Personally I'd leave it open for whenever mod_tile gets a maintainer.

Reply:
[I currently kind of maintain `mod_tile`](https://github.com/openstreetmap/mod_tile/issues/214). If someone more active joins, I'll be happy to cede, of course. In the meantime, I'm facilitating contributions, bug fixes, and cleaning up a few things. So if anyone actually has a need and wants to contribute, we can manage to bring this in. If no one comes forward, it is not my personal priority, so I won't work on it more.